### PR TITLE
Update camera specific input_agrs for ESP32-cam using ESPHome

### PR DESCRIPTION
**Pull request by MariusMueller**
Added "-c:v mjpeg" to MJPEG Cameras input_agrs in order to make a video stream work that comes from an ESP32-cam using ESPHome standard software

### CODE REVIEW

**Comment by NickM-27** [comment]
Looks like this is related to https://trac.ffmpeg.org/ticket/3679

File: docs/docs/configuration/camera_specific.md [hunk 1]
```diff
@@ -150,4 +150,4 @@ ffmpeg:
   output_args:
     record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -ar 44100 -c:a aac
     rtmp: -c:v copy -f flv -ar 44100 -c:a aac
-```
\ No newline at end of file
+```
```
Comment (NickM-27) [inline docs/docs/configuration/camera_specific.md:153]: What's different about this line? Seems like an unrelated change that should be ignored

**Comment by MariusMueller** [comment]
create new clea pull request
